packages/functional_widget/README.md

```diff
@@ -217,6 +217,20 @@ There are a few ways to do so:
   Widget example(int foo, String bar) => Container();
   ```
 
+- With parameters on the `@FunctionalWidget` decorator that will export the generated widget for private functions:
+
+  ```dart
+  @FunctionalWidget(public: true)
+  Widget _privateButExportedExample(int foo, String bar) => Container();
```

Comment:
I think I'd prefer have:

```dart
Widget __private(...) {...}

// generate 
class _Private extends Widget {}

Widget _public(...) {...}

// generate 
class Public extends Widget {}
```

Reply:
This would change the current generator's output.
If that's okay, we can do this also.

Right now, all _private widgets get converted to _Private.
__private will get converted to __Private

In the future, _private will be Private and
__private will be _Private.

This means removing the first underscore from function names by default if I do understand your example correctly?

Reply:
Indeed. It would be breaking, but I think this is alright